ROADMAP.md

```diff
@@ -72,6 +79,8 @@
 - Context
 - [Glossaries](https://github.com/machinetranslate/machinetranslate.org/issues/84)
 - [Controlled language](https://github.com/machinetranslate/machinetranslate.org/issues/85)
+- Data augmentation
+- Tuning
```

Comment:
What do you mean by this?

Reply:
Here I was especially thinking at tuning for PBMT (what was done with MERT, for example). But now that I re-think about it, this is maybe too PBMT-specific to be assigned a whole section here?

Reply:
Ah.  My instinct there would be to let it happen organically.  If the *Tuning* section of the SMT article gets too big, then it can be broken off to make it's own article.

Now you've raised the interesting question of how PBMT and SMT should be related.